l10n_it_fatturapa_out/models/partner.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Roberto Fichera <roberto.fichera@levelprime.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import fields, models, api, _
+from odoo.exceptions import ValidationError
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    max_invoice_in_xml = fields.Integer(
+        string='Max Num. Invoice in XML',
```

Comment:
Essendo una casella di selezione, meglio tenerla più compatta.

```suggestion
        string='Max Invoice # in XML',
```

Reply:
Ok